### DISCUSSION TITLE
Launcher does not work

### DISCUSSION THREAD

**Discussion by ITA50I** [serge-ivamov/USCLauncher#2] [Q&A]
When I pressed RUN button in launcher, it didn't work

so I tried to open use-game manually, but this happened.

/Applications/USCLauncher.app/Contents/Resources/unnamed-sdvx-clone/usc-game ; exit;
dyld[24992]: Library not loaded: /opt/homebrew/opt/libssh2/lib/libssh2.1.dylib
  Referenced from: <5BB7B65E-7DF6-3489-A328-AF67CFCE5D77> /Applications/USCLauncher.app/Contents/Resources/unnamed-sdvx-clone/usc-game
  Reason: tried: '/opt/homebrew/opt/libssh2/lib/libssh2.1.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/opt/libssh2/lib/libssh2.1.dylib' (no such file), '/opt/homebrew/opt/libssh2/lib/libssh2.1.dylib' (no such file), '/usr/local/lib/libssh2.1.dylib' (no such file), '/usr/lib/libssh2.1.dylib' (no such file, not in dyld cache)
zsh: abort      /Applications/USCLauncher.app/Contents/Resources/unnamed-sdvx-clone/usc-game

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

**Comment by serge-ivamov** [comment]
Hello,
my mistake - I did not include the openssh library in the app bundle.
You can install it yourself through brew. ( https://brew.sh/ https://formulae.brew.sh/formula/libssh2#default )
```bash
$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
$ brew install libssh2
```

**Reply by ITA50I** [reply]
THANK YOU SO MUCH

**Reply by ITA50I** [reply]
Is there any way to set keys?

**Reply by serge-ivamov** [reply]
Is it works now?
No, the game crashes when trying to save settings.
But you can customize the buttons under windows and manually edit the config. (Key_* strings)

**Comment by serge-ivamov** [comment]
The problem has been fixed in the latest release.
